src/main/java/com.springcore.ref/Test3.java

```diff
@@ -7,7 +7,7 @@
 import org.springframework.context.support.ClassPathXmlApplicationContext;
 
 public class Test3 {
```

Comment:
Invalid naming conventions. use proper name to indicate the class's functionality